.vscode/settings.json

```diff
@@ -13,8 +13,12 @@
 			}
 	],
 
+	"vim.hlsearch": false,
```

Comment:
My settings.json is objectively the best though... 😉 

Okay, okay, we should probably add it to .gitignore. :P